listeners/src/main/java/org/fao/geonet/listener/metadata/draft/DraftUtilities.java

```diff
@@ -157,6 +156,9 @@ public AbstractMetadata replaceMetadataWithDraft(AbstractMetadata md, AbstractMe
 
             Log.info(Geonet.DATA_MANAGER, "Record updated with draft contents: " + md.getId());
 
+            Log.info(Geonet.DATA_MANAGER, "Cleaning up draft record resources");
```

Comment:
I suggest that you include the metadata uuid and metadata id that is being cleaned up.  This can help a lot when debugging.
i.e.
"Cleaning up draft record resources for metadata '" + draft.getUuid() + "(" +draft.getId()  +")'"

Reply:
fixed